docker-compose.yml

```diff
@@ -7,6 +7,7 @@ services:
     environment:
       LOG_LEVEL: "DEBUG"
       API_BASE_PATH: "/api/"
+      EVENT_STORAGE_DSN: "redis://redis:6379?db=0&connection_timeout=5&minsize=5&maxsize=10"
```

Comment:
What do you think if we decrease the number of connections in the pool? I would propose to have 1-4 for development.

Reply:
Done